packages/client/src/containers/Path.container.ts

```diff
@@ -17,10 +16,11 @@ export const Path = createContainer(() => {
   // Fetch BOTH the path AND it's modules
```

Comment:
Can you update the comment?

Reply:
updated. Commit 4a12935. feat(api): modify according to first two comments